codebase/superdarn/src.lib/tk/radar.1.22/src/rprm.c

```diff
@@ -81,23 +82,74 @@ int RadarParmSetOriginTime(struct RadarParm *ptr,char *str) {
 
 }
 
+/*
+ * GetRSTVersion 
+ * Get the rst version from the .rst_version file 
+ * 
+ * return:
+ *  rst_version char * of the version number of RST 
+ *  if the version is not found then NULL is returned 
+ *  and error message appears. 
+ */
+char* GetRSTVersion() {
+
+  char *rst_path=NULL;
+  //fprintf(stderr, "rst_version\n");
+  char *rst_version = malloc(80);
+  char vname[256];
+  char buff[100];
+  // used the save the .rst_version file
+  FILE *vfp=NULL;
+
+  rst_path=getenv("RSTPATH");
+  strcpy(vname, rst_path);
+  strcat(vname, "/.rst.version");
+
+  vfp=fopen(vname, "r");
+
+  if (vfp != NULL) {
+    // read the .rst_version file 
+    while (fscanf(vfp,"%s",buff)==1); 
+    strcat(rst_version, " RST Version: ");
```

Comment:
Does it make sense to add some kind of delimiter between the command and the RST version number? No strong opinion here.
```suggestion
    strcat(rst_version, " -- RST Version: ");
```